packages/smooth_app/lib/main.dart

```diff
@@ -174,6 +174,9 @@ class SmoothApp extends StatelessWidget {
             svg,
             width: _navigationIconSize,
             height: _navigationIconSize,
+            color: context.watch<DarkThemeProvider>().darkTheme
```

Comment:
Isn't that `themeData.background`?

Reply:
Sure, I must be tired sorry

Reply:
Actually I used Theme.of(context).accentColor